modules/462-loki/openapi/config-values.yaml

```
@@ -0,0 +1,185 @@
+type: object
+properties:
+  effectiveStorageClass:
+    oneOf:
+      - type: string
+      - type: boolean
+    x-doc-default: 'false'
+  diskSizeGigabytes:
+    type: integer
+    description: |
+      Disk size for log store. PVC size if `effectiveStorageClass` set.
```

```suggestion
      Disk size for storing logs or the size of the PVC when specifying the [storageClass](#parameters-storageclass) parameters.
```
It seems that the effectiveStorageClass parameter have te be named as storageClass.